manifests/a/awwbees/BespokeSynth/1.0.0/awwbees.BespokeSynth.installer.yaml

```diff
@@ -10,7 +10,7 @@ Installers:
 - Architecture: x64
   InstallerType: msi
   Scope: machine
-  InstallerUrl: https://github.com/awwbees/BespokeSynth/releases/download/v1.0.0/Bespoke-Windows.msi
+  InstallerUrl: https://www.bespokesynth.com/builds/Bespoke-Windows.msi
```

Comment:
```suggestion
  InstallerUrl: https://github.com/BespokeSynth/BespokeSynth/releases/download/v1.0.0/Bespoke-Windows.msi
```

Reply:
Looks like it was moved to `BespokeSynth/BespokeSynth`.

The published release comes from @awwbees